frontend/src/Editor/Inspector/Components/Table/ProgramaticallyHandleToggleSwitch.jsx

```diff
@@ -18,28 +19,29 @@ export const ProgramaticallyHandleToggleSwitch = ({
   const param = { name: property };
   const definition = { value, fxActive: props.fxActive };
   const initialValue = definition?.value ?? `{{false}}`;
-
   const options = {};
   return (
-    <CodeHinter
-      enablePreview={true}
-      currentState={currentState}
-      initialValue={initialValue}
-      mode={options.mode}
-      theme={darkMode ? 'monokai' : options.theme}
-      lineWrapping={true}
-      onChange={(value) => callbackFunction(index, property, value)}
-      componentName={`widget/${component.name}::${label}`}
-      type={paramMeta.type}
-      paramName={param.name}
-      paramLabel={paramMeta.displayName}
-      fieldMeta={paramMeta}
-      onFxPress={(active) => {
-        callbackFunction(index, 'fxActive', active);
-      }}
-      fxActive={props?.fxActive ?? false}
-      component={component}
-      className="codehinter-default-input"
-    />
+    <div className={`mb-2 field ${options.className}`} onClick={(e) => e.stopPropagation()}>
+      <CodeHinter
+        enablePreview={true}
+        currentState={currentState}
+        initialValue={initialValue}
+        mode={options.mode}
+        theme={darkMode ? 'monokai' : options.theme}
+        lineWrapping={true}
+        onChange={(value) => callbackFunction(index, property, value)}
+        componentName={`widget/${component.component.name}::${param.name}`}
```

Comment:
I think we should check if the parent object is empty since we are pulling out a nested attribute
eg: `component?.component?.name` or use lodash `get` method